yew-components/src/select.rs

```diff
@@ -66,6 +66,15 @@ pub struct Props<T: Clone> {
     /// Options are available to choose.
     #[prop_or_default]
     pub options: Vec<T>,
+    /// Classes applied to the `<select>` tag
+    #[prop_or_default]
+    pub classes: String,
```

Comment:
Let's name this `class` to stay consistent with html elements. I think `yew-router` should rename to be consistent with yew